packages/blockchain-wallet-v4-frontend/src/hooks/useSelfCustodyCoinsBalances/useSelfCustodyCoinsBalances.tsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from 'react'
+import { useSelector } from 'react-redux'
+
+import { getBalanceSelector } from 'components/Balances/selectors'
+import { RootState } from 'data/rootReducer'
+
+import { CoinDataItem } from '.'
+
+export const useSelfCustodyCoinsBalances = () => {
+  const state = useSelector((state: RootState) => state)
+  const [coins, setCoins] = useState<CoinDataItem[] | null>(null)
+
+  const allowedChains = ['ETH', 'BTC', 'XLM', 'BCH', 'STX']
```

Comment:
Can you add a new list here:

https://github.com/blockchain/blockchain-wallet-v4-frontend/blob/development/packages/blockchain-wallet-v4/src/redux/data/coins/selectors.ts

const getSelfCustodyCoins = getDynamicSelfCustodyCoins().concat(getNonCustodialCoins())

and use that?